dev-docs/show-native-ads.md

```diff
@@ -114,7 +114,8 @@ pbjs.addAdUnits({
     mediaTypes: {
         native: {
             image: {
-                required: true
+                required: true,
+                sizes: [150, 50]
```

Comment:
looks like this adUnit has both adUnit.sizes as well as adUnit.mediaTypes.native.image.sizes and adUnit.mediaTypes.native.icon.sizes

Reply:
http://www.nooooooooooooooo.com

OK, updated in fe4679c to remove `adUnit.sizes` - thanks!